eap-latest-setup.md

```diff
@@ -209,233 +191,130 @@ in the labs.
-
-### Install Ansible
-Ansible currently comes from the EPEL repository.
-
-Install EPEL:
-
-    yum -y install \
-    http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
-
-Disable EPEL so that it is not accidentally used later:
+1. Install EPEL repo and then ansible
 
+    ```
+    yum -y install http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
     sed -i -e "s/^enabled=1/enabled=0/" /etc/yum.repos.d/epel.repo
-
-Install the packages for Ansible:
-
     yum -y --enablerepo=epel install ansible
+    ```
 
-### Generate SSH Keys
-Because of the way Ansible works, SSH key distribution is required. First,
-generate an SSH key on your master, where we will run Ansible:
-
-    ssh-keygen
-
-Do *not* use a password.
-
-### Distribute SSH Keys
-An easy way to distribute your SSH keys is by using a `bash` loop:
+1. Clone the Atomic Enterprise Ansible repo:
 
-    for host in ae-master.example.com ae-node1.example.com \
-    ae-node2.example.com; do ssh-copy-id -i ~/.ssh/id_rsa.pub \
-    $host; done
-
-Remember, if your FQDNs are different, you would have to modify the loop
-accordingly.
-
-### Clone the Ansible Repository
-The configuration files for the Ansible installer are currently available on
-Github. Clone the repository:
-
-    cd
+    ```
+    cd /root
     git clone https://github.com/projectatomic/atomic-enterprise-ansible.git
-    cd ~/atomic-enterprise-ansible
-
-### Configure Ansible
-Copy the staged Ansible configuration files to `/etc/ansible`:
-
-    /bin/cp -r ~/training/eap-latest/ansible/* /etc/ansible/
-
-### Modify Hosts
-If you are not using the "example.com" domain and the training example
-hostnames, modify `/etc/ansible/hosts` accordingly.
-
-Also, if you are using multiple NICs and will be trying to direct various
-traffic to different places, you will need to take a look at [Generic Cloud
-Install](#generic-cloud-install) to learn more about the syntax of Ansible's
-`hosts` file.
-
-### Run the Ansible Installer
-Now we can simply run the Ansible installer:
-
-    ansible-playbook ~/atomic-enterprise-ansible/playbooks/byo/config.yml
-
-If you looked at the Ansible hosts file, note that our master
-(ae-master.example.com) was present in both the `master` and the `node`
-section.
+    ```
+
+1.  Make sure (root) has an ssh key pair
+
+    ```
+    ssh-keygen # generally defaults are good
+    ```
+
+1. Copy the ssh public key to **all** nodes in the cluster
+
+    ```
+    for node in ae-master.example.com ae-node1.example.com ae-node2.example.com; do
+        ssh-copy-id ${node}
+    done
+    ```
+    Remember to replace the hostnames with your hostnames
+
+1. Edit the byo (bring your own) inventory file to include your hosts
+    ```
+    cd /root/atomic-enterprise-ansible
+    vi inventory/byo/hosts
+    ```
+    - \[masters\] section should contain only ae-master.example.com (or your hostname)
+    - \[nodes\] section should contain all of your nodes.
```

Comment:
Node zones need to be configured as well. At least section [Node Labels](https://github.com/eparis/atomic-enterprise-training/blob/ed90c5080d160dc79c618ee970db9a4c24d3fb7b/eap-latest-setup.md#node-labels) shows

    ae-node1.example.com    kubernetes.io/hostname=ae-node1.example.com,region=primary,zone=east     Ready
    ae-node2.example.com    kubernetes.io/hostname=ae-node2.example.com,region=primary,zone=west     Ready

Perhaps it would be easier to just replace whole `[nodes]` section with:
```
[nodes]
ae-master.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
ae-node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
ae-node2.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
```